### PR TITLE
4.21.2 Release

### PR DESCRIPTION
**Pull request by nuno-vieira**
### 🐞 Fixed
- Fix the Message List cell not updating when a Reaction of the same type was updated but from the current user [#2304](https://github.com/GetStream/stream-chat-swift/pull/2304)
- Fix the Message List cell not updating when the number of reactions of the same type changed [#2304](https://github.com/GetStream/stream-chat-swift/pull/2304)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # Upcoming
 
+# [4.21.2](https://github.com/GetStream/stream-chat-swift/releases/tag/4.21.2)
+_September 19, 2022_
+
+## StreamChatUI
+### 🐞 Fixed
+- Fix the Message List cell not updating when a Reaction of the same type was updated but from the current user [#2304](https://github.com/GetStream/stream-chat-swift/pull/2304)
```
Comment (polqf) [inline CHANGELOG.md:11]:
```suggestion
- Fix Message List cell not updating when an existing reaction of the same type was updated from the current user [#2304](https://github.com/GetStream/stream-chat-swift/pull/2304)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
...
+- Fix the Message List cell not updating when the number of reactions of the same type changed [#2304](https://github.com/GetStream/stream-chat-swift/pull/2304)
```
Comment (polqf) [inline CHANGELOG.md:12]:
```suggestion
- Fix Message List cell not updating when the number of reactions of the same type changed [#2304](https://github.com/GetStream/stream-chat-swift/pull/2304)
```

**Review by polqf — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (polqf) [reply CHANGELOG.md:12]: Also, I would not add 2 entries but just one. Maybe this last one

Comment (nuno-vieira) [reply CHANGELOG.md:12]: This is actually 2 fixes, not 1, so it makes sense to have both. There's 1 issue related to reaction counts not increasing, and another one, which is current user reactions not appearing as well

**Pull request merged**